### PR TITLE
list_of_events: imap-commands - Add cmd_input_name and update cmd_name

### PR DESCRIPTION
**Pull request by sirainen**
DOV-3785

### CODE REVIEW

File: source/admin_manual/list_of_events.rst [hunk 1]
```diff
@@ -812,10 +812,17 @@ IMAP command
 |                     |                                                      |
 |                     | .. versionadded:: v2.3.9                             |
 +---------------------+------------------------------------------------------+
-| cmd_name            | IMAP command name (e.g. ``FETCH``)                   |
+| cmd_name            | IMAP command name uppercased (e.g. ``FETCH``).       |
+|                     | Since v2.3.11 contains "unknown" for unknown command |
+|                     | names.                                               |
```
Comment (cmouse) [inline source/admin_manual/list_of_events.rst]: iirc there is `.. versionchanged: ` too.

Comment (sirainen) [reply source/admin_manual/list_of_events.rst]: Yeah. Now it looks like:

```
IMAP command name uppercased (e.g. FETCH).

New in version v2.3.9.

Changed in version v2.3.11: Contains “unknown” for unknown command names.
```

Comment (sirainen) [reply source/admin_manual/list_of_events.rst]: Actually, I'll remove quotes from "unknown" also and make it monospace.

File: source/admin_manual/list_of_events.rst [hunk 2]
```diff
@@ -812,9 +812,17 @@ IMAP command
...
-| cmd_name            | IMAP command name (e.g. ``FETCH``)                   |
+| cmd_name            | IMAP command name uppercased (e.g. ``FETCH``).       |
 |                     |                                                      |
 |                     | .. versionadded:: v2.3.9                             |
+|                     |                                                      |
+|                     | .. versionchanged:: v2.3.11 Contains ``unknown`` for |
+|                     |                     unknown command names.           |
++---------------------+------------------------------------------------------+
+| cmd_input_name      | IMAP command name exactly as sent (e.g. ``fetcH``).  |
+|                     | Also contains the name of unknown commands.          |
```
Comment (jeffpc) [inline source/admin_manual/list_of_events.rst]: I'd reword this a little:

IMAP command name exactly as sent (e.g., ``fetcH``) regardless of whether or not it is a valid.

Comment (sirainen) [reply source/admin_manual/list_of_events.rst]: I removed "a", i.e. "... it is valid." I was wondering whether to change to "it is a valid command." but maybe that's repeating the command too much.
